src/components/drawer/Swipeable.tsx

```diff
@@ -132,18 +135,22 @@ export default class Swipeable extends Component<PropType, StateType> {
     }
   }
 
-  // TODO: change to componentDidUpdate
-  UNSAFE_componentWillUpdate(props: PropType, state: StateType) {
+  componentDidUpdate(prevProps, prevState) {
+    this.prevProps = prevProps;
+    this.prevState = prevState;
```

Comment:
Why storing these? 
Why not just have the check here..

Reply:
I need to check after the first render

Reply:
See my other comment, this may be redundant.